pkg/controllers/dynakube/controller.go

```diff
@@ -331,6 +332,15 @@ func (controller *Controller) reconcileComponents(ctx context.Context, dynatrace
 		return err
 	}
 
+	log.Info("start reconciling monitored entities")
```

Comment:
what's our plan for these type of logs? Do they provide much benefit?

I'd expect a similar closing log as well that the reconcile finished successfully, if we have them, or only logs for interesting events during the reconcile, potentially with a logger name that provides more information.

Reply:
I do not have a strong opinion here, I can happily add another log line on this level that tells us that the reconciliation is done. Currently I am doing that on the level of the monitored entities reconciler, so I could do that on the higher level as well. For me it is just important to have some kind of entry point that indicates that reconciliation started, in order to make debugging somewhat easier.

maybe the others could share their opinion here as well.

Reply:
I would do it inside the monitored-entity reconciler, before it does the actual work, so it should only show up once for a given dynakube